### ISSUE TITLE
Nuke heat spike

### ISSUE THREAD

**Issue by M3tal-Warrior** [UmbraSpaceIndustries/MKS#1318]
Whenever I approach a mining platform of mine powered by a nuke reactor, the core heat is always maxed and the drills eat through the electricity buffers faster than a rat on fire. The miner itself has plenty of heat panels (4 large TCS for 8 Automated Industrial Strip Miners on 1170% load each), which never go above 50% load. Yet at load the reactors show a core temp of 1380K and drop to 9.9% efficiency. The AISMs themselves don't have this problem, they remain at 500K. 

While toying around I noticed the TCS don't matter much - the core temperature of all components doesn't rise or keeps dropping down even if I retract all of them. Is it possible to directly link the core overshoot temperatures to the panels?
Or maybe this whole subject can be prevented in a way similar to this:
1. Calculate max. produced heat of the whole vessel per second (heat_prod)
2. Calculate max. dissipated heat through TCS of the whole vessel per second (heat_diss)
3. if heat_prod < heat_diss; set all temperatures on every module to environment/target temperature, because this is roughly where they would be at the equilibrium
4. if heat_prod >= heat_diss; calculate overshoot temperatures (since efficiency and thus heat production declines with temperatures rising above target temperatures).

**Comment by BobPalmer** [comment]
The changes you are noting above would be stock changes not MKS ones.  Are you using any other mods that affect reactors (Heat Control, Near Future, etc.)?  And what version of KSP are you on?

**Comment by M3tal-Warrior** [comment]
Nope, no other mods tampering with the heat system. And it's 1.3 (latest version). 

Sorry this is the wrong site - does the dev team has a GitHub page to push the issue over there?
